src/Optimize/VariableSet.cpp

```diff
@@ -274,7 +274,7 @@ void VariableSet::print(std::ostream& os, int leftPadSpaces, bool printHeader)
        << setw(max_value_len) << NameAndValue[i].second << " " << setw(max_type_len) << ParameterType[i].second << " "
        << setw(max_recompute_len) << Recompute[i].second << " ";
 
-    os << std::defaultfloat;
+//    os << std::defaultfloat;
```

Comment:
Could you paste printout before and after this change? It helps us understanding the change.

Reply:
The only reason I commented this line out is because the compiler I am using doesn't seem to support this manipulator and I will get a compiler error 

the compiler I'm using  
intel/18.0.5/openmpi/1.10.1/bin/mpicc
intel/18.0.5/openmpi/1.10.1/bin/mpicxx

the error
/ascldap/users/sdpined/Desktop/qmcpack_pull_request_Ye/src/Optimize/VariableSet.cpp(277): error: namespace "std" has no member "defaultfloat"
      os << std::defaultfloat;

Reply:
Could you type "icc -v" and report the output here? I think it is probably due a too low gcc version. It seems passing our C++14 check.